webdriver/tests/get_timeouts/get.py

```diff
@@ -23,15 +23,6 @@ def test_get_timeouts(session):
     assert isinstance(value["pageLoad"], int)
 
 
-def test_get_default_timeouts(session):
```

Comment:
It does seem slightly unfortunate to lose the test coverage here, but AFAIK we can't have any sort of arguments to a fixture to opt-out of this…

Reply:
Oh, I missed to mention it here but we do have the exact same tests under `/new_session/timeouts.py`. So removing this duplicate doesn't make a difference!